### PR TITLE
Update "Use Origin Trials in Microsoft Edge" per new OT portal

### PR DESCRIPTION
**Pull request by leahmsft**
Motivation for this PR:
We're launching a new site for origin trials at [https://developer.microsoft.com/microsoft-edge/origin-trials](https://developer.microsoft.com/microsoft-edge/origin-trials) on 10/17 (tentative).  The old site [https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/](https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/) will still remain working, to support the Ad Selection API origin trial, until the new site supports that OT.

Rendered articles for review:
* **Use origin trials in Microsoft Edge**
   * `/origin-trials/index.md`
   * https://review.learn.microsoft.com/microsoft-edge/origin-trials/?branch=pr-en-us-3288
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/origin-trials/)
   * Changed title from "Use Origin Trials in Microsoft Edge".
   * Integrated Getting Started content from old portal.
      * Designed heading "Registering for an origin trial" to be target of an fwlink from the new OT site UX.
      * Designed heading "Using the origin trial token at your website" to be target of an fwlink from the new OT site UX.
      * Redesigned and integrated the headings from "Getting Started > Developer guidelines" to present high-value h2 headings in "In this article" nav.
      * Noted that the Ad Selection API OT still uses the old OT site.
      * Outline of headings:
```
# Use origin trials in Microsoft Edge
## (intro)
#### How the origin trial token works

## Registering for an origin trial

## Using the origin trial token at your website
#### Option 1: A \<meta\> tag within the \<head\> element of a webpage
#### Option 2: An HTTP header in your HTTP server response
#### Opting out of an origin trial experiment

## Performing feature detection and providing graceful fallback

## Testing an origin trial at `localhost`

## Renewing an origin trial token

## Duration of an origin trial
#### Scheduled expiration date
#### Early end

## Providing feedback about an origin trial

## See also
```

* **Sign up for the Ad Selection API** > **Provide feedback about the origin trial**
   * `/web-platform/ad-selection-api.md`
   * https://review.learn.microsoft.com/microsoft-edge/web-platform/ad-selection-api?branch=pr-en-us-3288#provide-feedback-about-the-origin-trial
   * [Planned live](https://learn.microsoft.com/microsoft-edge/web-platform/ad-selection-api#provide-feedback-about-the-origin-trial)
   * Added "Provide feedback about the origin trial" h2 section.
   * Linked to section from OT article.
* **Experimental features and origin trials for PWAs**
   * `/progressive-web-apps-chromium/how-to/origin-trials.md`
   * https://review.learn.microsoft.com/microsoft-edge/progressive-web-apps-chromium/how-to/origin-trials?branch=pr-en-us-3288
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/progressive-web-apps-chromium/how-to/origin-trials)
   * Changed title from "Experimental features and origin trials", to differentiate from the broader Edge-scoped article.
* **Test and automation in Microsoft Edge**
   * `test-and-automation.md`
   * Added missing link to Edge OTs article, in OTs section.
   * Updated OTs section/intro by copy/paste from latest Edge OTs article.
* **What's New in Progressive Web Apps**
   * `/whats-new/pwa.md`
   * https://review.learn.microsoft.com/microsoft-edge/progressive-web-apps-chromium/whats-new/pwa?branch=pr-en-us-3288
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/progressive-web-apps-chromium/whats-new/pwa)
   * Removed excess links to OT site where "origin trial" is mentioned.
* **Develop for the web with Microsoft Edge** > **Test and automation**
   * `develop-web-microsoft-edge.md`
   * https://review.learn.microsoft.com/en-us/microsoft-edge/develop-web-microsoft-edge?branch=pr-en-us-3288#test-and-automation
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/develop-web-microsoft-edge#test-and-automation)
   * Lowercased "origin trial".
* **Handle links to your Progressive Web App** > **Handle links from other origins by using scope extensions**
   * `/progressive-web-apps-chromium/how-to/handle-urls.md`
   * https://review.learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/handle-urls?branch=pr-en-us-3288#handle-links-from-other-origins-with-scope-extensions
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/progressive-web-apps-chromium/how-to/handle-urls#handle-links-from-other-origins-with-scope-extensions)
   * Rewrote section, to separate the desc from the "Notes:" OTs content; equivalent to converting an Alert (deprecated) to a specific heading.
   * Changed heading from "Handle links from other origins with scope extensions", per corp style guide.
   * Commented that the PWA OTs article lacks section about this PWA OT.
* Hub page
   * `~/index.yml`
   * https://review.learn.microsoft.com/microsoft-edge/?branch=pr-en-us-3288#microsoft-edge-for-developers
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/#microsoft-edge-for-developers)
   * In "Test and automation" card, put full title of articles, for clarity and consistency.
* Docset landing page
   * `/developer/index.yml`
   * https://review.learn.microsoft.com/microsoft-edge/developer/?branch=pr-en-us-3288
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/developer/)
   * In "Test and automation" card, put full title of articles, for clarity and consistency.
* TOC
   * `toc.yml`
   * https://review.learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/whats-new/pwa?branch=pr-en-us-3288
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/progressive-web-apps-chromium/whats-new/pwa)
   * Put full title of "What's New in Progressive Web Apps".
      * Limitations: When type "what's new" in "Filter by title", this article is still not listed, b/c too many DevTools articles, above it in TOC - evidence: What's New in DevTools 80 isn't listed; the final item listed is "Microsoft Edge 81".
   * Put full titles of "Handle foo" articles in TOC.
      * To clarify TOC entry for maintainers (& users).
      * For consistency.
      * Use `displayName:` as little as possible.  When fabricate a different, shortened TOC title, that extra line is required, to make full title appear in "Filter by title" UI.
* Global in repo
   * Lowercased "origin trial".
   * Changed old OT site url to new url, except for Ad Selection API OT.

AB#54425026

### CODE REVIEW

File: microsoft-edge/origin-trials/index.md [hunk 1]
```diff
@@ -13,12 +13,73 @@ You can use Origin Trials to try out experimental APIs on your own live site for
...
 
-To see a list of the available origin trials and register your origin to give them a try, see [Microsoft Edge Origin Trials](https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/).  To assess the suitability of using the experimental APIs on your website, check the minimum version requirements for Microsoft Edge, and the trial end date.
+To see a list of the available origin trials and register your origin to give them a try, see [Microsoft Edge Origin Trials](https://developer.microsoft.com/microsoft-edge/origin-trials).  To assess the suitability of using the experimental APIs on your website, check the minimum version requirements for Microsoft Edge, and the trial end date.
 
 You can also provide feedback to browser engineers and the web standards community about the design of the API, your use cases, or your experience using the APIs.
 
-For more information, see the **Getting Started** and **Developer Guidelines** sections in [Microsoft Edge Origin Trials](https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/).
+> [!IMPORTANT]
```
Comment (mikehoffms) [inline microsoft-edge/origin-trials/index.md:20]: Convert alert to standard specific heading.

https://review.learn.microsoft.com/en-us/help/platform/markdown-reference?branch=main#alerts-note-tip-important-caution-warning - "Avoid Note, Tip, and Important alerts—readers tend to skip over them. Placing that info directly into the article text is better."

i plan to push commit

File: microsoft-edge/origin-trials/index.md [hunk 2]
```diff
@@ -13,12 +13,73 @@ You can use Origin Trials to try out experimental APIs on your own live site for
...
+To see a list of the available origin trials and register your origin to give them a try, see [Microsoft Edge Origin Trials](https://developer.microsoft.com/microsoft-edge/origin-trials).  To assess the suitability of using the experimental APIs on your website, check the minimum version requirements for Microsoft Edge, and the trial end date.
 
 You can also provide feedback to browser engineers and the web standards community about the design of the API, your use cases, or your experience using the APIs.
 
-For more information, see the **Getting Started** and **Developer Guidelines** sections in [Microsoft Edge Origin Trials](https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/).
+> [!IMPORTANT] 
+> If you're interested in the Ad Selection API origin trial, please register [here](https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/). [Learn more about sign-up.](../web-platform/ad-selection-api.md)
```
Comment (mikehoffms) [inline microsoft-edge/origin-trials/index.md:21]:
```suggestion
> If you're interested in the Ad Selection API origin trial, register [here](https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/). [Learn more about sign-up.](../web-platform/ad-selection-api.md)
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please - avoid 'please'

i plan to push commit

Comment (mikehoffms) [inline microsoft-edge/origin-trials/index.md:21]: change [here] link text to dest title per corp style

i plan to push commit

dest title = [Microsoft Edge Origin Trials]
```suggestion
> To sign up for the Ad Selection API origin trial, go to [Microsoft Edge Origin Trials]https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/). [Learn more about sign-up.](../web-platform/ad-selection-api.md)
```

additional retagging/reformat is required; i'll push commit

File: microsoft-edge/origin-trials/index.md [hunk 3]
```diff
@@ -13,12 +13,73 @@ You can use Origin Trials to try out experimental APIs on your own live site for
...
 
+## Getting Started
+
+To register for an origin trial, 
+1. [Go to Microsoft Edge Origin Trials](https://developer.microsoft.com/microsoft-edge/origin-trials) and select an active trial.
+
+2. Complete the registration form and click the **Submit** button to get a valid token. 
+
+To use an origin trial token, you can use one of these two options: 
+1. Using a `<meta>` Tag: Add an origin-trial tag to the head of any page.
```
Comment (mikehoffms) [inline microsoft-edge/origin-trials/index.md:31]: change this numbered list to unordered list bc not sequential

i plan to push commit

File: microsoft-edge/origin-trials/index.md [hunk 4]
```diff
@@ -1,26 +1,172 @@
...
+
+
+<!-- ------------------------------ -->
+#### Duration of an origin trial
+
+
+<!-- ---------- -->
+###### A scheduled expiration date
+
+Once you have registered for a trial, your token will be valid until the expiration date that's listed on the origin trial's informational card.  After this date, the token will no longer function, and the feature that's associated with the trial will be considered ready to ship, or will be brought back<!-- todo: clarify "brought back" - removed from OT portal? --> for further development based on developer feedback.
```
Comment (mikehoffms) [inline microsoft-edge/origin-trials/index.md:135]:
```suggestion
Once you have registered for a trial, your token will be valid until the expiration date that's listed on the origin trial's informational card.  After this date, the token will no longer function, and the feature that's associated with the trial will be considered ready to ship, or the feature will be withheld for further development based on developer feedback. 
```
* "brought back" - https://learn.microsoft.com/en-us/style-guide/global-communications/writing-tips - "Avoid idioms, colloquial expressions, and culture-specific references. They can be confusing for non-native English speakers and hard to localize."

Comment (mikehoffms) [reply microsoft-edge/origin-trials/index.md:135]: result:
```
After you register for a trial, your token remains valid until the expiration date that's listed on the origin trial's informational card.  After this date, the token will no longer function, and the feature that's associated with the trial will be considered ready to ship, or the feature will be withheld for further development.
```
@leahmsft - alts:
* withheld
* brought back

File: microsoft-edge/origin-trials/index.md [hunk 5]
```diff
@@ -1,26 +1,172 @@
...
+<!-- ------------------------------ -->
+#### Best practices for experimental feature implementation
+
+Treat origin trials as experimental.  Provide graceful fallback mechanisms in case the origin trial token expires, or in case the browser feature team ends the origin trial.
+
+For best practices for feature detection, see [Implementing feature detection](https://developer.mozilla.org/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection) at MDN.
 
 
 <!-- ====================================================================== -->
 ## See also
```
Comment (mikehoffms) [inline microsoft-edge/origin-trials/index.md:163]: added 
* [Get started with origin trials](https://developer.chrome.com/docs/web-platform/origin-trials) - Chrome docs.
ok?  @leahmsft

File: microsoft-edge/origin-trials/index.md
Comment (mikehoffms) [inline microsoft-edge/origin-trials/index.md:1]: in portal & article, Init Caps or lowercase?
According to this Chrome docs team article, "origin trial" is normally/usually lowercase:
https://developer.chrome.com/docs/web-platform/origin-trials

rendered article:
https://review.learn.microsoft.com/microsoft-edge/origin-trials/?branch=pr-en-us-3288

@leahmsft

Comment (mikehoffms) [reply microsoft-edge/origin-trials/index.md:1]: keeping as lowercased, for review

File: microsoft-edge/origin-trials/index.md (hunk 5 above)
Comment (mikehoffms) [reply microsoft-edge/origin-trials/index.md:181]: keeping link, for review

File: microsoft-edge/origin-trials/index.md [hunk 6]
```diff
@@ -1,26 +1,172 @@
...
+
+<!-- ------------------------------ -->
+#### Subdomains, URL paths, and query parameters
+
+
+<!-- ---------- -->
+###### Subdomains
+
+Origin trial tokens can be configured to support any of the following:
+* A single domain, without supporting its subdomains.  For example, register `https://example.com`, and in the **Subdomain Support** section, select the **No** option button.
```
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/origin-trials/index.md]: updated ui verbiage per new portal

**Review by mikehoffms — APPROVED** [review]
LGTM - Writer/Editor

File: microsoft-edge/origin-trials/index.md [hunk 7]
```diff
@@ -1,26 +1,191 @@
...
+
+
+<!-- ====================================================================== -->
+## Allowed origins
+
+Edge origin trials only support SSL-enabled domains. For localhost testing of a feature, please use the appropriate Edge Feature Flag located in `edge://flags`. In the future, we may publish public tokens that are valid for localhost testing.
+
+
+<!-- ====================================================================== -->
+## Subdomains, URL paths, and query parameters
```
Comment (mikehoffms) [inline microsoft-edge/origin-trials/index.md:121]: move up to bottom of Register section, where this info is needed

File: microsoft-edge/origin-trials/index.md [hunk 8]
```diff
@@ -1,26 +1,191 @@
...
+
+1. Click **Renew**.  A fresh token is sent to you.
+
+1. Copy the new token and use it on your site; see [How to use an origin trial token](#how-to-use-the-origin-trial-token), above.
+
+
+<!-- ====================================================================== -->
+## Allowed origins
+
+Edge origin trials only support SSL-enabled domains. For localhost testing of a feature, please use the appropriate Edge Feature Flag located in `edge://flags`. In the future, we may publish public tokens that are valid for localhost testing.
```
Comment (mikehoffms) [inline microsoft-edge/origin-trials/index.md:117]: Deleting "In the future, we may publish public tokens that are valid for localhost testing."
, even though it's present at https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/
The statement shouldn't be present there, unless someone got special exception.

Docs should cover how the system presently works - not how they worked in the past or might work in the future.  Avoid promises.
https://developers.google.com/style/timeless-documentation
@leahmsft - i can probably find equivalent guideline in our corp style guide; says use present tense...

File: microsoft-edge/origin-trials/index.md (hunk 7 above)
Comment (mikehoffms) [reply microsoft-edge/origin-trials/index.md]: i integrated/moved info about how to fill in the form, into the steps for Registering
@leahmsft

File: microsoft-edge/origin-trials/index.md [hunk 9]
```diff
@@ -1,26 +1,204 @@
...
+<!-- ------------------------------ -->
+#### Opt out of an origin trial experiment
+
+To opt out of an experiment (origin trial) that you're enrolled in, remove the origin trial token from the `<meta>` tag or from the server response headers.
+
+
+<!-- ====================================================================== -->
+## Renew an origin trial token
+
+Origin trial tokens expire in 6 weeks, by default<!-- todo: strike ", by default"? -->.  When the origin trial token expires, you must renew the token, which means generating a fresh, new token for this origin trial.  You can renew the token before it expires.
```
Comment (mikehoffms) [inline microsoft-edge/origin-trials/index.md:129]: strike "by default"?  raises unresolved questions.  if we keep "by default", need to add hint or explanation of what scenario would be other than the default.  
are we saying that some tokens expire in 4 weeks or 8 weeks, depending on feature team?
@leahmsft

**Review by mikehoffms — APPROVED** [review]
LGTM - Writer/Editor

File: microsoft-edge/origin-trials/index.md
Comment (mikehoffms) [inline microsoft-edge/origin-trials/index.md:1]: global in repo: update all Origin Trial content (text & pngs)
@leahmsft @captainbrosset 
i'll do this and then re-approve PR; will enter Changes Requested to cancel previous approval

**Review by mikehoffms — CHANGES_REQUESTED** [review]
global in repo: update all Origin Trial content (text & pngs)

**Review by mikehoffms — APPROVED** [review]
LGTM - Writer/Editor

File: microsoft-edge/progressive-web-apps-chromium/how-to/origin-trials.md [hunk 10]
```diff
@@ -8,37 +8,62 @@ ms.service: microsoft-edge
 ms.subservice: pwa
 ms.date: 10/20/2023
 ---
-# Experimental features and origin trials
+# Experimental features and origin trials for PWAs
 
-Some of the PWA features in Microsoft Edge are still experimental.  Experimental features can be used in either of two ways:
+<!-- todo: add scope extensions OT, per
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/how-to/origin-trials.md:13]: Add a section about scope extensions, in PWA OTs article.
@captainbrosset

The article section
https://review.learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/handle-urls?branch=pr-en-us-3288#handle-links-from-other-origins-by-using-scope-extensions
is about scope extensions (an OT), yet the PWA OTs article doesn't mention "scope" or "extension":
https://review.learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/origin-trials?branch=pr-en-us-3288

This OT is Active, at Chrome status site.
Added a hidden todo comment in both articles.

File: microsoft-edge/test-and-automation/test-and-automation.md
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/test-and-automation/test-and-automation.md:1]: closing b/c entered fresh non-Outdated PR Comment

File: microsoft-edge/progressive-web-apps-chromium/how-to/origin-trials.md (hunk 10 above)
Comment (captainbrosset) [reply microsoft-edge/progressive-web-apps-chromium/how-to/origin-trials.md:13]: I don't think we should document the active OTs on the PWA OTs article. The OT portal will have the up to date information already. If we try to duplicate that information on our own OT-related articles, we're bound to get out of date eventually.

Mentioning specific OTs on specific feature pages, such as the handle-urls article seems ok to me, but also bound to get out of date at some point. But I wouldn't want to duplicate this information any further.

Comment (mikehoffms) [reply microsoft-edge/progressive-web-apps-chromium/how-to/origin-trials.md:13]: I will remove todo comment from both articles.  I don't recall my context except that it was .md POV in VS Code, maybe i should've checked rendered docs POV to remember what kind of OT content in which kinds of articles eg What's New PWAs.

Comment (mikehoffms) [reply microsoft-edge/progressive-web-apps-chromium/how-to/origin-trials.md:13]: thx for checking on this comment, i didn't intend to recommend covering a specific pwa OT in the general article about PWA ot's.  @captainbrosset

**Review by leahmsft — COMMENTED** [review]
lgtm-pm

**Pull request merged**